### PR TITLE
Updated example for SVM

### PR DESCRIPTION
**Pull request by scaactk**
The return value of SVM algorithm is float data, inorder to be consistent with plabels read from csv. We need to binarize the result

### CODE REVIEW

File: examples/svm_batch.py [hunk 1]
```diff
@@ -60,6 +60,16 @@ def main(readcsv=read_csv, method='defaultDense'):
 
 if __name__ == "__main__":
     (predict_result, plabels) = main()
+    """
+    The return value of SVM algorithm is float data, inorder to be consistent with plabels read from csv
+    We need to binarize the result
+    """
+    for i in range(len(predict_result.prediction)):
```
Comment (PetrovKP) [inline examples/svm_batch.py:67]: I think better return decision function and result of classification. See example for GPU: https://github.com/IntelPython/daal4py/blob/master/examples/sycl/svm_batch.py#L104

Comment (scaactk) [reply examples/svm_batch.py]: yeah, the suggestion is nice and I have updated the code and submitted a new pull request

**Comment by PetrovKP** [comment]
@scaactk  Good! Thanks, but this upgraded is failed CI:
```
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/CB/conda-bld/daal4py_1592817974732/test_tmp/tests/test_examples.py", line 115, in test_svm_batch
    (predict_result, _) = self.call(ex)
  File "/usr/share/miniconda/envs/CB/conda-bld/daal4py_1592817974732/test_tmp/tests/test_examples.py", line 216, in call
    return ex.main(readcsv=pd_read_csv)
  File "/usr/share/miniconda/envs/CB/conda-bld/daal4py_1592817974732/test_tmp/examples/svm_batch.py", line 62, in main
    return (decision_result, predict_lables, plabels)
NameError: name 'predict_lables' is not defined
```

File: examples/svm_batch.py [hunk 2]
```diff
@@ -55,11 +55,17 @@ def main(readcsv=read_csv, method='defaultDense'):
     # Prediction result provides prediction
     assert(predict_result.prediction.shape == (pdata.shape[0], 1))
 
-    return (predict_result, plabels)
+    # result of classification
+    decision_result = predict_result.prediction
+    predict_labels = np.where(decision_result >=0, 1, -1)
+
+    return (decision_result, predict_lables, plabels)
```
Comment (PetrovKP) [inline examples/svm_batch.py:62]:
```suggestion
    return (decision_result, predict_labels, plabels)
```

Comment (scaactk) [reply examples/svm_batch.py]: Sorry, the variable name has been corrected

**Comment by PetrovKP** [comment]
We use examples as small tests. So we broke the tests a bit. It is necessary to add a return value in the line:
https://github.com/IntelPython/daal4py/blob/master/tests/test_examples.py#L115
For example you can make:
```python
   (decision_result, _, _) = self.call(ex)
   self.assertTrue(np.absolute(decision_result - testdata).max() < np.absolute(decision_result.max() - decision_result.min()) * 0.05)

```

**Comment by scaactk** [comment]
@PetrovKP  Thank you very much for guiding me step by step~

**Pull request merged**